src/main/java/greencity/controller/PlaceCommentController.java

```diff
@@ -0,0 +1,74 @@
+package greencity.controller;
+
+import greencity.constant.ErrorMessage;
+import greencity.constant.HttpStatuses;
+import greencity.dto.comment.AddCommentDto;
+import greencity.dto.comment.CommentReturnDto;
+import greencity.entity.Place;
+import greencity.entity.User;
+import greencity.entity.enums.UserStatus;
+import greencity.exception.NotFoundException;
+import greencity.exception.UserBlockedException;
+import greencity.service.PlaceCommentService;
+import greencity.service.PlaceService;
+import greencity.service.UserService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.security.Principal;
+import javax.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@AllArgsConstructor
+public class PlaceCommentController {
+    /**
+     * Autowired CommentService instance.
+     */
+    private PlaceCommentService placeCommentService;
+    private UserService userService;
+    private PlaceService placeService;
+
+
+    /**
+     * Method witch save comment by Place Id.
+     *
+     * @param placeId       Id of place to witch related comment.
+     * @param addCommentDto DTO with contain data od Comment.
+     * @return CommentDTO
+     */
+    @ApiOperation(value = "Add comment.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = CommentReturnDto.class),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+    })
+    @PostMapping("/place/{placeId}/comments")
+    public ResponseEntity save(@PathVariable Long placeId,
+                               @Valid @RequestBody AddCommentDto addCommentDto, Principal principal) {
+        User user = userService.findByEmail(principal.getName())
+            .orElseThrow(() -> new NotFoundException(ErrorMessage.USER_NOT_FOUND_BY_EMAIL + principal.getName()));
+        if (user.getUserStatus().equals(UserStatus.BLOCKED)) {
+            throw new UserBlockedException(ErrorMessage.USER_HAS_BLOCKED_STATUS);
+        }
```

Comment:
I am not sure, but I think all logic we should place at the service

Reply:
Also you can do something like this 
![image](https://user-images.githubusercontent.com/21316358/66814046-9945a580-ef3e-11e9-988e-4170cd737d95.png)
instead of getting "principal" as a param

Reply:
Principal is removed, this logic should located here as blocked user will not get access to service layer